### PR TITLE
feat: use bns-2 branch of connect

### PR DESCRIPTION
**Pull request by hstove**
Uses version of `@stacks/connect` from https://github.com/blockstack/stacks-wallet-web/pull/1103

### CODE REVIEW

File: src/auth.js [hunk 1]
```diff
@@ -9,13 +9,15 @@ export function authenticate() {
   showConnect({
     appDetails: {
       name: 'Todos',
-      icon: window.location.origin + '/logo.svg',
+      icon: window.location.origin + '/logo.png',
     },
     redirectTo: '/',
+    authOrigin: 'https://pr-1103.app.stacks.engineering/',
```
Comment (markmhx) [inline src/auth.js:15]: Do we want to hardcode this URL? Presumably that will mess things up upon deploy to todos.blockstack.org

**Review by markmhx — CHANGES_REQUESTED** [review]

File: src/auth.js (hunk 1 above)
Comment (markmhx) [reply src/auth.js:15]: Awaiting registrar on mainnet

**Comment by markmhx** [comment]
It appears this PR should go back into development to update the `authOrigin` value above to `app.blockstack.org` if I'm not mistaken.

**Comment by markmhx** [comment]
@fbwoolf to review and close if irrelevant now

**Comment by fbwoolf** [comment]
I believe this is outdated am going to close. I don't see an existing branch in connect related to `bns-2`.

**Pull request closed without merging**